test/gurka/test_gtfs.cc

```diff
@@ -215,19 +249,19 @@ TEST(GtfsExample, WriteGtfs) {
     .tuesday = CalendarAvailability::Available, .wednesday = CalendarAvailability::Available,
     .thursday = CalendarAvailability::Available, .friday = CalendarAvailability::Available,
     .saturday = CalendarAvailability::Available, .sunday = CalendarAvailability::Available,
-    .start_date = Date(2022, 1, 31), .end_date = Date(2023, 1, 31),
+    .start_date = Date(2023, 1, 31), .end_date = Date(2024, 1, 31),
```

Comment:
should do smth smarter here too soon (well, before january 2024;)). get the `now` date and offset a bit. careful since the tile creation date is also significant for the validity checks